frame/composable-support/src/validation.rs

```diff
@@ -65,17 +65,48 @@
 //! }
 //! ```
 
-use core::marker::PhantomData;
+use core::{fmt, marker::PhantomData};
 use scale_info::TypeInfo;
 use sp_runtime::DispatchError;
 
 /// Black box that embbed the validated value.
-#[derive(Default, Copy, Clone, PartialEq, Eq, Debug, TypeInfo)]
+#[derive(Default, Copy, Clone)]
```

Comment:
you can do that to default where T: Default . So Validated<T,_> can be default iff T is Default

Reply:
Take a look at 74095f779fe4b4221bbe24061e86d8249ec73354, I think I implemented this as you described

Reply:
hm, may be u said  something stupid. may be derive and derive_more do all these things. if traget is traited, than trait is for wrapper.  and probably infernce will work like with hashmap, but we have no example

Reply:
Reverted changes here 00187d3786d302f87f11c3415823c1ede0a943b0